frontend/src/components/CreateMemo/page.tsx

```diff
@@ -109,20 +65,19 @@ const CreateMemo = ({}) => {
     };
 
     try {
-      const res = await MemoController.create_memo(userID, memoToCreate);
-      // console.log(res)
-      const newMemoId = res._id; // TODO
+      const createdMemo = await MemoController.create_memo(currentUser.uid, memoToCreate);
+      const newMemoId = createdMemo._id; // TODO
 
       // Update the memos array in userData with the new memo ID
       let newUserData: any = userData;
       newUserData.memos = [...userData.memos, newMemoId];
 
       setUserData(newUserData);
```

Comment:
Shouldnt need to set user data upon creation of memo. Page will navigate to dashboard upon success. On failure of `UserController.update_user `, the UserData state on the create memo page should not be affected.

Reply:
just realized this has revealed a bug and has to do with a problem with both the logic on the frontend and design of our backend services.

see #57 but also #58